lambda_calc/parser.py

```diff
@@ -10,7 +10,7 @@ def lexeme(p: Parser) -> Parser: return p << ws
 
 
 lambda_ = lexeme(regex(r'[λ\\L]')).desc('lambda')
-var = lexeme(regex(r"[a-z]")).map(Var).desc('variable')
+var = lexeme(regex(r"[a-z](?:')?|[a-z]''")).map(Var).desc('variable')
```

Comment:
even though it's unlikely that there will be more than 2 `'`s but let's just do `r'[a-z]\'*'` to make the parser more forgiving. everything else looks good!

Reply:
sounds good, done